cypress/e2e/tests/migration/applicationinventory/analysis/source_analysis_without_creds.test.ts

```diff
@@ -74,6 +74,25 @@ describe(["@tier1"], "Source Analysis without credentials", () => {
         TaskManager.verifyStatus(application.name, succeeded);
     });
 
+    // Automates Bug https://issues.redhat.com/browse/MTA-3440
+    it("Source analysis on bookserver app with EAP8 target", function () {
+        application = new Analysis(
+            getRandomApplicationData("bookserverApp", {
+                sourceData: this.appData["bookserver-app"],
+            }),
+            getRandomAnalysisData(this.analysisData["EAP8_bookserverapp"])
+        );
+        cy.wait(2 * SEC);
+        Application.open();
+        application.create();
+        applicationsList.push(application);
+        cy.wait("@getApplication");
+        cy.wait(2000);
+        application.analyze();
+        checkSuccessAlert(infoAlertMessage, `Submitted for analysis`);
+        application.verifyAnalysisStatus("Completed");
```

Comment:
@kpunwatk Please check , when you are passing "Completed" In model it is checking if text contains "Completed" . That means it can "Complete with Errors" and still mark test as Pass.